index.html

```diff
@@ -58,22 +94,26 @@ <h4>I am a Software Developer</h4>
                     <img class="medium" src="./assets/medium-icon.png" alt="medium">
                 </a>
             </li>
-        </ul>       
+        </ul>
     </section>
     <section class="works">
         <div class="recent-works">
             <h2>My Recent Works</h2>
             <hr class="border">
+            <div class="desktop-works">
             <img src="assets/Img Placeholder.svg" alt="yoga stretch">
+            <div class="desktop-projects">
             <h3>Multi-Post Stories</h3>
-            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
```

Comment:
Your web page responding very well to different sizes. But let's solve the small text overlaying issue here to make the page more responsive.
<img width="1010" alt="Screenshot 2023-01-20 at 8 45 42 AM" src="https://user-images.githubusercontent.com/25887253/213630120-ac6603f8-e7c5-47ad-9714-fc17fe64fb72.png">